lib/elixir/test/elixir/enum_test.exs

```diff
@@ -47,6 +48,14 @@ defmodule EnumTest do
     assert Enum.at([2, 4, 6], -4) == nil
   end
 
+  test "chunk/3" do
+    enumerable = [1, 2, 3, 4, 5]
+
+    capture_io(fn ->
+      assert Enum.chunk(enumerable, 2, 1) == Enum.chunk(enumerable, 2, 1, :discard)
```

Comment:
So `Enum.chunk(enumerable, 2, 1, :discard)` was never officially supported. So I would match on the actual result and also add a test for `Enum.chunk(enumerable, 2, 1, nil)`.

Reply:
sure, no problem